sensorDashboard.mst

```diff
@@ -0,0 +1,231 @@
+        }
+
+        .dropdown,
+        .dropdown-toggle,
+        .dropdown-menu {
+            width: 100%;	
+        }
+
+        .nav-link.dropdown-item {
+            color: rgb(88, 88, 91);
+            overflow: hidden; 
+            white-space: nowrap; 
+            text-overflow: ellipsis;
+        }
+
+        .no-locations {
+            opacity: .75;
+        }
+
+        .btn-logout, .btn-logout > a {
+            width: auto;
+            text-decoration: none;
+        }    
+
+        #navbar-brand {
+            font-size: 24px;
+            color: rgb(247, 249, 250);
+        }
+
+        .content-area {
+            background-color: rgb(247, 249, 250);
+            color: rgb(88, 88, 91);
+        }
+
+        .content-wrapper {
+            margin-left: 7%;
+            margin-right: 7%;
+            margin-top: 5%;
+        }
+
+        .table-header {
+            font-weight: 600;
+        }
+
+        .table-data {
+            font-weight: 400;
+        }
+
+        div.content-wrapper>table {
+            table-layout: fixed;
+            max-width: 300px;
+            border-collapse: collapse;
+            border-spacing: 5px 5px;
+            margin-right: auto;
+        }
+
+        th, td {
+            text-align: left;
+            height: auto;
+            width: auto;
+            white-space: pre-wrap;
+            word-wrap: break-word;
+        }
+
+        th a{
+            text-decoration: none;
+            color: rgb(88, 88, 91);
+        }
+
+        th a:hover {
+            text-decoration: none;
+        }
+
+        .table>thead>tr>th,
+        .table>tbody>tr>th,
+        .table>tbody>tr>td {
+            border: none;
+            width: auto;
+        }
+
+        @media screen and (max-width: 770px) {
+
+            .navbar, .btn-logout, h3 {
+                font-size: 1em; 
+                font-weight: 900;
+            }
+
+            #navbar-brand {
+                font-size: 4.5vw;
+                font-weight: 700;
+            }
+
+            .nav-item {
+                text-align: right;
+            }
+
+            .navbar-toggler {
+                border: none;
+            }
+
+            .dropdown-menu {
+                background-color: rgb(0, 184, 184);
+                border: none;
+                padding-right: 3%;
+            }
+
+            .nav-link.dropdown-item {
+                color: rgb(247, 249, 250);
+                text-align: right;
+            }
+
+            .nav-link.dropdown-item:hover {
+                color: white;
+                background-color: rgb(0, 200, 200);
+            }
+
+            .dropdown-menu.show {
+                padding-top: 0%;
+            }
+
+            .btn-logout {
+                padding-right: 0%;
+            }
+
+            table {
+                font-size: 1.8vw;
+            }
+        }
+
+    </style>
+</head>
+
+<body>
+    <nav class="navbar navbar-expand-md">
```

Comment:
Is there a way to factor out the navbar (and any other repeated elements) into separate template files so that it can be used by both `locationsDashboard` and `sensorDashboard`?

Reply:
yeeessssss this is done now you are a genius 😅 I've been hating how long these are. I also separated out the CSS into one file each and while there is a lot of repetition between them it keeps them out of the way for the time being. I didn't think that was a great use of time this second as it would involve even more partial files and then it's just getting a bit ridiculous imo 😲 I mainly just wanted them out of the `<head>` so the file is readable without 30 years of scrolling, maybe I can pare everything down once I'm done adding further dashboards to see what stays consistent across it all?